mPowerSDK/Startup/APHAppDelegate.m

```diff
@@ -37,6 +37,7 @@
 #import "APHDataKeys.h"
 #import "APHLocalization.h"
 
+       NSString *const kDailySurveyIdentifier                       = @"3-APHMoodSurvey-7259AC18-D711-47A6-ADBD-6CFCECDED1DF";
```

Comment:
The rest of these are listed in APHDataKeys. Could you put this one there as well? Thanks.